### PR TITLE
[NEW ARTICEL] Virtualbox

### PR DESCRIPTION
**Pull request by ShowMeYourSkil**
Good afternoon @mtorials ,

This is my new articel about the Oracle Virtualbox. I've check the text about speeling mistakes.
If you find something else please write me a comment or edit my file.

See you soon!

### CODE REVIEW

**Comment by mtorials** [comment]
Thank you for spell checking this time ;)! Could you maybe add something about what to do next / how to boot an ISO image and where to download it and so on?

**Comment by ShowMeYourSkil** [comment]
> Thank you for spell checking this time ;)! Could you maybe add something about what to do next / how to boot an ISO image and where to download it and so on?

See my stap list. Click on the button type and select linux. Now click on the button version and select your favorite ISO. For example: Ubuntu or Redhat

In my next articel i want to work with ansible (maybe i should thinking about what my next step is :D).

**Comment by ShowMeYourSkil** [comment]
> Thank you for spell checking this time ;)! Could you maybe add something about what to do next / how to boot an ISO image and where to download it and so on?

In my next articel i want to work with ansible. Support your blog mp4 videos or only markdown documents?

**Comment by ShowMeYourSkil** [comment]
> > Thank you for spell checking this time ;)! Could you maybe add something about what to do next / how to boot an ISO image and where to download it and so on?
> 
> See my stap list. Click on the button type and select linux. Now click on the button version and select your favorite ISO. For example: Ubuntu or Redhat

In my next articel i want to work with ansible (maybe i should thinking about what my next step is :D).

**Comment by mtorials** [comment]
> See my stap list. Click on the button type and select linux. Now click on the button version and select your favorite ISO. For example: Ubuntu or Redhat

Yes thats right but you never say how to start an ISO image. Just where you select the OS type or am I missing something?

**Comment by ShowMeYourSkil** [comment]
> > See my stap list. Click on the button type and select linux. Now click on the button version and select your favorite ISO. For example: Ubuntu or Redhat
> 
> Yes thats right but you never say how to start an ISO image. Just where you select the OS type or am I missing something?

Its realy realy easy. The virtual box is very clearly and simple, but do you want that i add this in my article?

**Comment by mtorials** [comment]
Yes I think this would make it more complete.

**Comment by ShowMeYourSkil** [comment]
> Yes I think this would make it more complete.

Thx ;) okey

**Comment by ShowMeYourSkil** [comment]
> Yes I think this would make it more complete.

@mtorials now i have finished your wish.

**Comment by ShowMeYourSkil** [comment]
> Yes I think this would make it more complete.

!!! I have add more files to this pr. I have update your README and have add a pull request template for you.!!!

File: content/posts/virtualbox.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Open the Virtualbox
+2. Click the button `new` to create a new machine
+3. Now you must set a Name, a folder, the type and the version
+    - In my case i use the type `Linux`
+    - For the version can you now set the Linux system. In my case I use `Ubuntu 64bit` 
+4. Now you must set the Memory size
+5. Congratulations! Now you have your own virtual machine.
+
+# How to start a Virtualmachine
```
Comment (mtorials) [inline content/posts/virtualbox.md:47]: Where is the ISO part?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* [ ]  Other
+
+# Description
+
+<!-- Please describe briefly and in summary your pull request -->
+
+## Information
+<!-- Other additional information? -->
+
+> The Blog Owner @mtorials want to check your Pull Request. Please wait a few hours or days. He becomes to comment your Pull Request or Merge him.
```
Comment (mtorials) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: just leave this lien out

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Pull Request Description
+
+* [ ]  New Blog entry
+* [ ]  Correction Blog entry
+* [ ]  Other
+
+# Description
+
+<!-- Please describe briefly and in summary your pull request -->
+
```
Comment (mtorials) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: please add the license part (see CONTRIBUTING.md)

File: README.md [hunk 4]
```diff
@@ -1,16 +1,43 @@
...
+## Write your own articles with markdown
+![image](https://user-images.githubusercontent.com/39274150/111983370-59f38f00-8b0a-11eb-9b6b-377c71e16d35.png)
+
+# How to write a article?
+
+Do you want make a article! Nice! Please read THIS file when you're new and do not know how.
+
+# Netifly
+
+mt32.net work with the deploy platform `Netifly`. Netifly can deploy and build web apps based on github workflows.
```
Comment (mtorials) [inline README.md]: Not true anymore. I will update the README myself accordingly in the next few days

**Review by mtorials — CHANGES_REQUESTED** [review]

**Comment by mtorials** [comment]
Also, can you split this into two PRs? One for the post and one for the PR Templ? I will update the README myself.

**Comment by ShowMeYourSkil** [comment]
> Also, can you split this into two PRs? One for the post and one for the PR Templ? I will update the README myself.

Jep wait a hour :D

File: content/posts/virtualbox.md (hunk 1 above)
Comment (ShowMeYourSkil) [reply content/posts/virtualbox.md:47]: Step 3.2 

>3. Now you must set a Name, a folder, the type and the version
>  - In my case i use the type `Linux`
> - For the version can you now set the Linux system. In my case I use `Ubuntu 64bit`

**Comment by ShowMeYourSkil** [comment]
See my new PR

**Comment by ShowMeYourSkil** [comment]
> Also, can you split this into two PRs? One for the post and one for the PR Templ? I will update the README myself.

@mtorials how can i split my PRS?

**Comment by ShowMeYourSkil** [comment]
Bye bye

**Pull request closed without merging**